aws-sessionstore-dynamodb.gemspec

```diff
@@ -13,6 +13,6 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_dependency 'aws-sdk-v1'
-  spec.add_dependency 'rack', '~> 1.0'
+  spec.add_dependency 'aws-sdk'
```

Comment:
Should still keep a major version lock on this, such as `'~> 2.0'`.

Both because this in theory would allow a V1 dependency which would then mysteriously fail, and because if there's ever a V3, we don't want this to hard fail.